### DISCUSSION TITLE
Me resizewindowpixel dispatcher is on strike

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [hyprwm/Hyprland#1338] [Q&A] [answered]
The following does nothing for me, although it returns "OK". The PID is correct. What I am missing?

```
hyprctl dispatch resizewindowpixel exact 500 400, pid:5767
```

**Comment by raffaelemancuso** [comment]
Me pin dispatcher is on strike as well. They formed a dispatcher union to ask for a wage raise.

```
❯ hyprctl dispatch pin
Usage: hyprctl dispatch <dispatcher> <arg>
            Execute a hyprland keybind dispatcher with the given argument
```

**Reply by raffaelemancuso** [reply]
ok, for the `pin` dispatcher, it looks like an argument is required, so you need to put whatever argument after it:

```
hyprctl dispatch pin WHATEVER
```

**Comment by vaxerski** [comment, marked as answer]
`hyprctl dispatch resizewindowpixel exact 500 400,pid:5767`

mind the removed space after `,`

**Reply by raffaelemancuso** [reply]
yes it works after removing the space :)
